inst/www/shiny-semantic-calendar.js

```diff
@@ -0,0 +1,111 @@
+// Convert a number to a string with leading zeros
+function padZeros(n, digits) {
+  var str = n.toString();
+  while (str.length < digits)
+    str = "0" + str;
+  return str;
+}
+
+// Given a Date object, return a string in yyyy-mm-dd format, using the
+// UTC date. This may be a day off from the date in the local time zone.
+function formatDateUTC(date) {
+  if (date instanceof Date) {
+    return date.getUTCFullYear() + '-' +
```

Comment:
@ashbaldry May I ask you what was the reason for using UTC date? I'm thinking of replacing the functions with `getFullYear`, `getMonth` and `getDate` (functions that are used in fomantic-ui documentation). This way I can omit the difference between shiny input and received value in the shiny server.

Reply:
Honestly I took what was happening in the shiny calendar input, and applied the same sort of functionality for this input. Don't see any reason not to use the Fomantic-UI functionality